src/Sylius/Bundle/ShopBundle/Twig/Component/Product/PriceComponent.php

```diff
@@ -30,50 +33,58 @@ class PriceComponent
 
     public ProductVariant $variant;
 
+    #[ExposeInTemplate]
+    public ?string $price = null;
+
+    #[ExposeInTemplate(name: 'original_price')]
+    public ?string $originalPrice = null;
+
+    #[ExposeInTemplate(name: 'has_discount')]
+    public bool $hasDiscount = false;
```

Comment:
Why is there a missing name declaration here? Are the following defined to enforce snake case?
```suggestion
    #[ExposeInTemplate(name: 'price')]
    public ?string $price = null;

    #[ExposeInTemplate(name: 'original_price')]
    public ?string $originalPrice = null;
    ...
```